app/pages/TransactionPreview.tsx

```diff
@@ -91,6 +92,7 @@ export const TransactionPreview = ({
     account: `${accountIdentifier} • ${getInstitutionNameByCode(institution, supportedInstitutions)}`,
     description: memo || "N/A",
     network: selectedNetwork.chain.name,
+    providerId: providerId || "N/A",
```

Comment:
providerId cannot be "N/A". it either has a valid value or the field isn't included at all

Reply:
Yeah makes sense